addOns/ascanrules/src/main/java/org/zaproxy/zap/extension/ascanrules/httputils/HtmlContextAnalyser.java

```diff
@@ -200,7 +200,7 @@ public static Map<String, Map<String, String>> parseTag(String input) {
      * Function to parse HTML attributes string. Used specifically for cases where jericho's parsing
      * for internal tags fails.
      *
-     * @param input a String containing attributes to be parsed
+     * @param attr a String containing attributes to be parsed
```

Comment:
Oops sorry that was my fault

Reply:
it's attr now. I changed it as soon as I saw it